client/lib/signup/step-actions/index.js

```diff
@@ -349,17 +350,20 @@ function getDIFMLiteCartItemFromDependencies( dependencies ) {
 		displayPhone,
 		displayAddress,
 	} = dependencies;
+
+	const socialProfiles = getSocialProfiles( reduxStore.getState() );
```

Comment:
Maybe we can have a difm lite selector which transforms our state tree to a flat data structure of fields to be submitted. This way the step-actions module does not need to know about the UI separated sections of the state tree,

Reply:
Good idea. For now, I'll remove this change from the PR, and we can continue this part in a follow-up PR.